tachikoma-test-client/src/main/kotlin/SendTemplateEmail.kt

```diff
@@ -0,0 +1,78 @@
+import com.google.protobuf.Struct
+import com.google.protobuf.Value
+import com.google.protobuf.util.JsonFormat
+import com.sourceforgery.tachikoma.grpc.frontend.NamedEmail
+import com.sourceforgery.tachikoma.grpc.frontend.maildelivery.EmailRecipient
+import com.sourceforgery.tachikoma.grpc.frontend.maildelivery.MailDeliveryServiceGrpc
+import com.sourceforgery.tachikoma.grpc.frontend.maildelivery.OutgoingEmail
+import com.sourceforgery.tachikoma.grpc.frontend.maildelivery.TemplateBody
+import com.sourceforgery.tachikoma.grpc.frontend.maildelivery.TemplateEngine
+import io.grpc.ManagedChannelBuilder
+import java.time.Instant
+
+fun main(args: Array<String>) {
+    val channel = ManagedChannelBuilder.forAddress("localhost", 8070)
+            .usePlaintext(true)
+            .build()
+
+    val stub = MailDeliveryServiceGrpc.newBlockingStub(channel)
+
+    val template = """
+    <div class="entry">
+        <h1>{{title}}</h1>
+        <div class="body">
+            <div>
+            {{body}}
+            </div>
+            <ul>
+            {{#items}}
+                <li><strong>{{.}}</strong></li>
+            {{/items}}
+            </ul>
+        </div>
+    </div>
+"""
```

Comment:
Just a friendly reminder that there are things like https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/trim-margin.html in Kotlin. It's fine as it is now.